stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/usecase/SendMessageImpl.kt

```diff
@@ -25,6 +25,9 @@ internal class SendMessageImpl(private val domainImpl: ChatDomainImpl) : SendMes
 
         val channelController = domainImpl.channel(cid)
         return CoroutineCall(domainImpl.scope) {
+            if (message.replyMessageId != null) {
+                channelController.replyMessage(null)
```

Comment:
Because we don't need to keep reply state. Imagine you decide to make reply to some message, you select this one, write your reply, and click on the send button. Logically replied state has been dismissed and you don't want to see reply state anymore

Reply:
Okay I understand now. For some reason, I thought that `ReplyMessage` use case is going to send a reply, but it's only job is to update `LiveData` right? If yes, maybe it would be a good idea to change its name to something more self explanatory? e.g. `SetReplyMessage` or `ShowReplyMessage`

Reply:
Yeah, I think you're right. A good name can help you to understand things more easily

Reply:
Changed to `SetMessageForReply`